f600233d1b154d507b782f84604bcb12/update/sys_script_include_5d76f38e970611106fa8b4b3f153afc4.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.Request</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>ATAT class that helps make general request to external resources (EDS, CSS, etc.).</description>
+        <name>Request</name>
+        <script><![CDATA[var Request = Class.create();
+Request.prototype = {
+    initialize: function() {
+		// methods
+		this.GET = "GET";
+		this.POST = "POST";
+		this.PATCH = "PATCH";
+		this.DELETE = "DELETE";
+		
+		// mutual auths
+		this.JKS = "jks443"; // CSS
+		this.MAUTH = "mauth"; // EDA/PDS
```

Comment:
The Protocol Profile is really just a port/cert pair, so we should be defining them based on that rather than arbitrary names like `mauth` (the default in SNOW examples). Since EDA & CSS listen on 443, they should use `jks443`, whereas EDMS is on 8443, so it will need to use `jks8443`.  We could make `jks443` the default for `mutualAuth` if we want.

Reply:
Good explanation, thanks @jeffsegal-ccpo

Reply:
@jeffsegal-ccpo Okay, that makes sense, I wasn't really sure what was happening with the names so I just used the ones we had defined already.  Does the below make sense?

```javascript
    // mutual auths
    this.JKS_443 = "jks443"; // CSS, EDA/PDS
    this.JKS_8443 = "jks8443"; // EDMS
```

```javascript
    request.setMutualAuth(req.mutualAuth || this.JKS_443);
    // remove below
    // if (req.mutualAuth) {
    //	request.setMutualAuth(req.mutualAuth);	
    // }
```

Reply:
Sure that seems reasonable.